sql/schema.sql

```diff
@@ -0,0 +1,56 @@
+CREATE DATABASE capstone;
+
+CREATE TABLE genres (
+    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE music_albums (
+    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(100) NOT NULL,
+    publish_date DATE NOT NULL,
+    archived BOOLEAN,
+    on_spotify BOOLEAN NOT NULL,
+    genre_id INT,
+    FOREIGN KEY (genre_id) REFERENCES genres(id)
+);
+
+-- Create a table for books
+CREATE TABLE books(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  publisher VARCHAR(255),
+  book_title VARCHAR(255),
+  cover_state VARCHAR(255),
+  published_date DATE,
+  FOREIGN KEY(id) REFERENCES item(id)
+);
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/164895032-a71f05b7-bd13-486c-ba47-e43730a772d7.png)

- [ ] Make sure also you create the item table as you are referencing it.